scenarios/aws/fakeintake/fakeintake.go

```diff
@@ -201,8 +205,10 @@ func fargateSvcLB(e aws.Environment, namer namer.Namer, taskDef *awsxEcs.Fargate
 
 	fi.Scheme = "https"
 	fi.Port = httpsPort
-	fi.Host = pulumi.String(host).ToStringOutput()
-	fi.URL = pulumi.String(fi.Scheme + "://" + host).ToStringOutput()
+	fi.Host = host
+	fi.URL = host.ApplyT(func(host string) string {
+		return fi.Scheme + "://" + host
+	}).(pulumi.StringOutput)
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
What about using `pulumi.Sprintf` instead?

Reply:
Ah yes indeed I did know I could use that with a `pulumi.StringOutput` . I though it was used to change a classic `string` into `pulumi.String`